### PR TITLE
background: use __len__ for chunk length, minor style fixes

### PR DESCRIPTION
**Pull request by pkulev**
* Use __len__ for providing length of inner list
    This is common python approach. len calls __len__ of countable object.
* Use valid variable names in test instead of P and S, remove unused import.

**Closes issue(s):** [  ]

**Description of the changes:**

  * What's new:
  * Fixes:
  * Refactoring:

**Reviewers:** [ @taptap ]

**Task list:**

- [x] submit pull request
- [x] review
- [x] testing
- [x] fixes after review, squashing, rebasing to master

### CODE REVIEW

File: xoinvader/background.py [hunk 1]
```diff
@@ -32,24 +29,22 @@ def name(self):
...
-        self._length += 1
 
     def __getitem__(self, index):
         return self._lines[index]
 
     def __setitem__(self, index, item):
         self._lines[index] = item
 
+    def __len__(self):
+        return len(self._lines)
```
Comment (taptap) [inline xoinvader/background.py]: O(1) -> O(n)
I don't see why this is necessary.

File: xoinvader/background.py [hunk 2]
```diff
@@ -93,7 +88,7 @@ def load_chunks(filename, trim_width=None):
                 chunks.append(current_chunk)
                 names.append(name)
                 continue
-            elif not current_chunk:
+            elif current_chunk is None:
```
Comment (taptap) [inline xoinvader/background.py]: Please do style fixes in separate commits.

File: xoinvader/tests/test_background.py [hunk 3]
```diff
@@ -113,16 +114,16 @@ def test_background():
     assert b._advance_chunk(1) == "!@#"
     assert b._advance_chunk(1) == "   "
 
-    P, S = b.get_render_data()
```
Comment (taptap) [inline xoinvader/tests/test_background.py]: Please move all style changes in separate commit.

File: xoinvader/background.py (hunk 1 above)
Comment (pkulev) [inline xoinvader/background.py]: Why do you think that len take O(n)?
len calls PySIZE macro.
https://github.com/python/cpython/blob/9530f485920ebb1404d4adaf24e6f3f7ddbe9622/Objects/listobject.c#L397
It defines here: https://github.com/python/cpython/blob/d6a5199175066149c73f62ee020a3161e504e1cf/Include/object.h#L119
And

File: xoinvader/background.py (hunk 2 above)
Comment (pkulev) [inline xoinvader/background.py]: This is not style fix, this is related to upper changes.

File: xoinvader/background.py (hunk 1 above)
Comment (taptap) [inline xoinvader/background.py]: Okay. Didn't know that.

File: xoinvader/tests/test_background.py (hunk 3 above)
Comment (pkulev) [inline xoinvader/tests/test_background.py]: Done.

File: xoinvader/background.py (hunk 2 above)
Comment (taptap) [inline xoinvader/background.py]: Got it. Write about it in the commit message then. I mean that that is actually `if bool(current_chunk)` and `bool` calls `__len__` method if it's present and so on.

Anyway, checking for `None` should happen as `foo is None`, and this is primarily style issue.

Comment (pkulev) [inline xoinvader/background.py]: Why my commit message does not confirm?

Comment (taptap) [inline xoinvader/background.py]: It does. 
I just replied before I read new commits. Sorry for the confusion.
